src/node/in_memory.rs

```diff
@@ -369,8 +368,7 @@ impl<S: std::fmt::Debug + ForkSource> InMemoryNodeInner<S> {
             previous_batch_hash: None,
             number: L1BatchNumber::from(block_ctx.batch),
             timestamp: block_ctx.timestamp,
-            l1_gas_price: self.l1_gas_price,
-            fair_l2_gas_price: L2_GAS_PRICE,
+            fee_input: compute_batch_fee_model_input(self.l1_gas_price, 1.0, 1.0),
```

Comment:
Should we move `1.0` values to `const` values `L1_GAS_PRICE_SCALE_FACTOR ` and `L1_GAS_PRICE_SCALE_FACTOR` at the top of the file?